borg/helpers.py

```diff
@@ -44,13 +44,27 @@ def have_cython():
     import msgpack
 
 
+# return codes returned by borg command
+# when borg is killed by signal N, rc = 128 + N
+EXIT_SUCCESS = 0  # everything done, no problems
+EXIT_WARNING = 1  # reached normal end of operation, but there were issues
+EXIT_ERROR = 2  # terminated abruptly, did not reach end of operation
```

Comment:
no, currently "Error" exceptions are defined where needed.

what I am thinking about implementing at some time is a "constants" module that just has all these constants.